app/views/layouts/application.html.erb

```diff
@@ -37,31 +38,51 @@
 <%= javascript_include_tag "application" %>
 
 </head>
-<body>
-  <div class="container">
+
+<header> 
+  <div class="beta-span">
+    <a href="http://goo.gl/forms/Wt7iMAaN8a"><h4>We're in Beta. Help us improve this app! Click here to submit your feedback.</h4></a>
+  </div>
+  <div class= "header-container">
     <div class="row">
-     <span class="beta-span col-xs-12">
-      <a href="http://goo.gl/forms/Wt7iMAaN8a"><h4>We're in Beta. Help us improve this app! Click here to submit your feedback.</h4></a>
-    </span>
-    <div class="col-xs-12">
-      <a href="<%= root_path %>">
-        <%= image_tag 'pet-alerts-logo.png', class: 'logo' %>
-      </a>
+      <div class="col-xs-12">
+        <a href="<%= root_path %>" class= 'logo'>
+          <h1 class="main-title"> Pet Alerts </h1>
+        </a>
+      </div>
     </div>
+  </div>
+</header>
+
+<body>
+
+  <% [ :success, :danger, :info ].each do |msg_type| %>
+  <% if flash[msg_type] %>
+  <div class="alert alert-<%= msg_type %>" role="alert"><%= raw flash[msg_type] %></div>
+  <% end %>
+  <% end %>
+
+  <%= yield %>
+
+
+</div> <!-- /container -->
 
-    <% [ :success, :danger, :info ].each do |msg_type| %>
-    <% if flash[msg_type] %>
-    <div class="alert alert-<%= msg_type %>" role="alert"><%= raw flash[msg_type] %></div>
-    <% end %>
-    <% end %>
+<footer>
+  <div class= "container">
+    <div class= "wide-layout">
+      <div class= "unsubscribe">
+       <p>Pet Alerts sends you text or email notifications each time a new pet matching your search is listed.</p>
+       <div class="main-title-layout"><div class="unsubscribe-button text-center"><%= link_to 'Unsubscribe', unsubscribe_url %></div>
```

Comment:
@Tallys You have two divs opening on this line, but only one closing - really needs consistency.